x-pack/plugins/fleet/server/services/security/message_signing_service.test.ts

```diff
@@ -153,7 +153,8 @@ describe('MessageSigningService', () => {
         MESSAGE_SIGNING_KEYS_SAVED_OBJECT_TYPE,
         keyPairObj.id,
         {
-          passphrase: expect.any(String),
+          ...keyPairObj.attributes,
```

Comment:
Does adding this make the test case fail without your fix? Want to make sure there's test coverage added for this bug as part of this PR.